apps/fishing-map/features/reports/events/VGREvents.tsx

```diff
@@ -87,8 +88,8 @@ function VGREvents() {
             t('vesselGroup.summaryEvents', {
               defaultValue:
                 '<strong>{{vessels}} vessels</strong> from <strong>{{flags}} flags</strong> had <strong>{{activityQuantity}} {{activityUnit}}</strong> globally between <strong>{{start}}</strong> and <strong>{{end}}</strong>',
-              vessels: vessels?.length,
-              flags: vesselFlags,
+              vessels: formatI18nNumber(vessels?.length || 0),
```

Comment:
Shouldn't we use here `formatI18nNumber(getVesselGroupVesselsCount(vesselGroup))` like in vessel report name?

Reply:
This is not the same. This shows the vessels with events for the current time range, not all vessels in a group. I renamed the variable to make things clearer